### DISCUSSION TITLE
Customer with a list of orders in the authorization response

### DISCUSSION THREAD

**Discussion by maxpanakov** [medusajs/medusa#2606] [Q&A] [answered]
Do we really need full list of orders ? this loads the request.  Can it be optional ? 
![image](https://user-images.githubusercontent.com/9987863/201855154-bb57f8d6-59c5-46f8-b747-a03dee97e109.png)

**Comment by srindom** [comment, marked as answer]
Think this is a great point! There are a couple of things that would need to be done here, however: 

1. Support `fields` and `relations` params on the customer endpoints
2. Set `defaultRelations`, `defaultFields` for `/store/customers/*` endpoints.
3. Ensure that `/store/customers/me/orders` works as expected.

Would you be interested in opening a PR on this?
